ui/storybook/package.json

```diff
@@ -9,7 +9,9 @@
     "storybook:serve": "npx http-server ./storybook-static",
     "lint": "eslint src --ext .ts,.tsx",
     "lint:fix": "npm run lint -- --fix",
-    "type-check": "tsc --noEmit"
+    "type-check": "tsc --noEmit",
+    "happo": "happo",
+    "happo-ci-github-actions": "HAPPO_COMMAND=../node_modules/happo.io/build/cli.js npx happo-ci-github-actions"
```

Comment:
I can't comment in a package.json file or I would. We need to explicitly tell happo where to look for that executable because of some weirdness with our monorepo setup that their script doesn't quite account for.

Reply:
Also added an explanation in the README.